src/mcts/search.cc

```diff
@@ -1302,7 +1302,8 @@ void SearchWorker::DoBackupUpdateSingleNode(
     n->FinalizeScoreUpdate(v, d, node_to_process.multivisit);
 
     // Convert parents to terminals except the root or those already converted.
-    can_convert = can_convert && p != search_->root_node_ && !p->IsTerminal();
+    can_convert =
```

Comment:
Probably cleaner to add `if (!p) break;` in a line above.
Although I'm a bit lost how it happened that the loop condition is not enough anymore..

Reply:
Before #822, there were no uses of the parent within the loop.